Sources/Preferences/Settings.swift

```diff
@@ -5,7 +5,35 @@ public enum Settings {}
 
 // TODO: Remove in the next major version.
 // Preserve backwards compatibility.
+@available(*, deprecated, renamed: "Settings")
```

Comment:
@sindresorhus With all the changes, I wonder if merely removing these old names would be better since function signatures also changed? 🤔

Reply:
I think it would be useful to keep these as Xcode can then help with the migration.